README.md

```diff
@@ -40,15 +41,36 @@ fn main() {
 ```
 
 ### Decoding
+
 ```rust
 use blurhash::decode;
 
 let pixels = decode("LBAdAqof00WCqZj[PDay0.WB}pof", 50, 50, 1.0);
 ```
 
+To decode into an `ImageBuffer`, add the `image` feature flag, then do:
```

Comment:
```suggestion
Blurhash-rs supports directly decoding into an `ImageBuffer` of the `image` crate. Add the `image` feature flag, then do:
```

I would rephrase it (like for instance above) to emphasize that this uses the `image` crate, and that said dependency is optional.